src/fetch.js

```diff
@@ -182,6 +182,17 @@ const fetchCsData = async (url, config, query) => {
       branch: config?.branch ? config.branch : 'main',
     },
   };
+  // Check if config has the key: enableEarlyAccess,IF YES, it's an array of strings, and it's not empty
+  if (
+    config?.enableEarlyAccessValue &&
+    config?.enableEarlyAccessKey &&
+    config?.enableEarlyAccessKey !== '' &&
```

Comment:
seems confusing `enableEarlyAccessValue` and  `enableEarlyAccessKey` is it two different key or key/value ?